examples/examples_test.go

```diff
@@ -115,6 +115,14 @@ func TestExampleObjectSchemas(t *testing.T) {
 			"redis-master-service":   &api.Service{},
 			"redis-slave-service":    &api.Service{},
 		},
+		"../examples/guestbook/guestbook_ns": {
```

Comment:
I suggest creating `v1beta3` subdirectories in each example, with a short README.md just stating that it's a v1beta3 version of the example, and explaining that v1beta3 needs to be enabled.

In a couple weeks we'll promote these to the top level and eliminate the v1beta1 examples.